### PR TITLE
completed implementation of checkShip function

### PR DESCRIPTION
**Pull request by Krishnasaianumandla**

### CODE REVIEW

**Review by Pragya28 — COMMENTED** [review]
You can make the function smaller by checking not equal to EMPTY UNCLICKED and directly returning False.

Good Work.

File: battleship.py [hunk 1]
```diff
@@ -80,15 +92,22 @@ def createShip():
...
 def checkShip(grid, ship):
-    return
+    count=0
+    for i in ship:
+        if grid[i[0]][i[1]] == EMPTY_UNCLICKED:
+            count+=1
+            if count==3:
+               return True
+        else:
+            return False
```
Comment (Pragya28) [inline battleship.py:102]: You can reduce the number of lines in the function by checking the not equal condition and returning False, and returning True on completion of the loop.

Comment (Pragya28) [reply battleship.py:102]: @Krishnasaianumandla please make the requested changes.

Comment (Krishnasaianumandla) [reply battleship.py]: Resolved the problems.

File: battleship.py [hunk 2]
```diff
@@ -92,14 +92,10 @@ def createShip():
...
-    count=0
     for i in ship:
-        if grid[i[0]][i[1]] == EMPTY_UNCLICKED:
-            count+=1
-            if count==3:
-               return True
-        else:
+        if grid[i[0]][i[1]] != EMPTY_UNCLICKED:
             return False
+    return True
```
Comment (Pragya28) [inline battleship.py]: Good work.

**Review by Pragya28 — COMMENTED** [review]
Good work.

**Review by manasa133 — APPROVED** [review]

**Pull request merged**